src/sls_core.js

```diff
@@ -320,6 +320,60 @@ function getFunctionByName(functionName) {
   return new Function(`return ${functionName};`)();
 }
 
+/**
+ * Checks if the given URL is a valid image URL.
+ *
+ * @param {string} url The URL to check.
+ * @return {boolean} Whether the URL is a valid image URL.
+ */
+function isValidImageUrl(url) {
+  return url.startsWith('http://') || url.startsWith('https://');
```

Comment:
Are you planning on doing something like below and checking the mime-type of the asset located at the URL? Otherwise this is really just "startsWithHttp" and doesn't really validate the string is a URL.

App Script doesn't let you do a HEAD call, but you could get the URL and check the content type. Something like 
`return  (UrlFetchApp.fetch(url).getHeaders())['Content-Type'].startsWith('image')`

might not be worth it?

Reply:
I could change it to startsWithHttpOrHttps, as this is the requirement of the SlidesAPI function, and then consider expanding to test whether it's an image

Reply:
You could just add a todo and an FR. That signals intent, but allows you to just leave this as is.